.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
+ignore = E203, E266, E501, W503, F403, F401
+max-line-length = 79
+max-complexity = 18
+select = B,C,E,F,W,T4,B9
```

Comment:
I think some of these recommendations are outdated. According to [this article](https://mitelman.engineering/blog/python-best-practice/automating-python-best-practices-for-a-new-project/) you really only need `extend-ignore = E203`. You can keep the `max-line-length` one if you also set the black one to be 79.